packages/astro/components/ViewTransitions.astro

```diff
@@ -220,15 +222,23 @@ const { fallback = 'animate' } = Astro.props as Props;
 		links.length && (await Promise.all(links));
 
 		if (fallback === 'animate') {
+			let isAnimating = false;
+			addEventListener('animationstart', () => (isAnimating = true), { once: true });
+
 			// Trigger the animations
 			document.documentElement.dataset.astroTransitionFallback = 'old';
-			const finished = Promise.all(document.getAnimations().map(a => a.finished));
```

Comment:
did you remove this change on purpose?

Reply:
Whoops, no. No idea how that got removed!